README.md

```diff
@@ -1 +1,6 @@
-# mobile-menu
+# curriculum-cr-training-mobile-menu
+
+
```

Comment:
Please include the getting started instructions as well as a screenshot of the website